POLYGLOT_README.md

```diff
@@ -112,7 +112,7 @@ Polyglot Piranha can be used as a python library or as a command line tool.
 
 `pip install polyglot-piranha`
 
-Currently, we support one simple API (`run_piranha_cli`) that wraps the command line usage of Polyglot Piranha. We believe this makes it easy to incorporate Piranha in *"pipelining"*.
+Currently, we support one simple API (`execute_piranha`) that wraps the command line usage of Polyglot Piranha. We believe this makes it easy to incorporate Piranha in *"pipelining"*.
```

Comment:
Just to clarify: 

1) Does this mean there is a separate CLI for Polyglot Piranha, unrelated to `run_piranha_cli`? Where `run_piranha_cli` is just a confusingly named alternative python API, now deprecated in favor of `execute_piranha` and has nothing to do with running Piranha as a CLI tool directly?

Or:

2) Are we actually dropping all support from running Polyglot Piranha as a command line tool directly? (i.e. it's now meant to be used exclusively as a Rust/Python library, and CLI tools will be built using that library).

This docs make me think the first interpretation is correct, but I want to be sure.